en/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md

```diff
@@ -0,0 +1,137 @@
+
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+
+## Configuration
+
+The Plugin-Pack is designed to monitor resources based on one host per account.
+Adding a host into Centreon, link it to the template named *App-Cisco-Ssms-Restapi-custom*. 
+Once the template applied, some Macros have to be configured:
+
+| Mandatory   | Nom                    | Description                                                                |
+| :---------- | :--------------------- | :------------------------------------------------------------------------- |
+| X           | APIPORT                | Port used. Default is 443                                                  |
+| X           | APIPROTO               | Protocol used. Default is https                                            |
+| X           | CLIENTID               | Client ID to access to the API.                                            |
+| X           | CLIENTSECRET           | Client Secret to access to the API.                                        |
+|             | APIEXTRAOPTIONS        | Any extra option you may want to add to the command                        |
+
+
+## FAQ
+
+### How to check in the CLI that the configuration is OK and what are the main options for ?
+
+Once the Plugin installed, log into your poller using the *centreon-engine* user account and test by running the following command (Parameters such as ```account``` or ```client-id```have to be adjusted):
+
+```bash
+/usr/lib/centreon/plugins/centreon_cisco_ssms_restapi.pl \
+    --plugin=apps::cisco::ssms::restapi::plugin \
+	--mode=licenses \
+	--hostname='myipaddress' \
+	--client-id='myapiclientid' \
+	--client-secret='myapiclientsecret' \
+	--account='1234abc-56de-78fg-90hi-1234abcdefg' \
+	--filter-counters='status' \
+	--filter-license-name='mylicence'
+	--critical-license-status='%{status} !~ /in compliance/i' \
+	--verbose
+	
+```
+
+The command above gets the status of a Cisco SSMS licenses (```--mode=licenses```) named *mylicense* (```--filter-licence-name='mylicence'```). 
+This license belongs to account *1234abc-56de-78fg-90hi-1234abcdefg* (```--account='1234abc-56de-78fg-90hi-1234abcdefg'```). 
+
+This command will trigger a CRITICAL alarm if it contains the word *in compliance* (```--critical-license-status='%{status} !~ /in compliance/i'``) 
+
+All the options that can be used with this plugin can be found over the ```--help``` command:
+
+```/usr/lib/centreon/plugins//centreon_cisco_ssms_restapi.pl --plugin=apps::cisco::ssms::restapi::plugin --mode=licences --help```
+
+
+### How do I remove the *count* perfdatas if I want to filter on just one application ?
+
+The Plugin adds the count of objects by default. This can be useless if the objects are filtered with the ```--filter-license-name``` parameter.
+Therefore, these useless perfdatas can be omitted by adding a perfdata filter : ```--filter-perfdata='^$'```.
```

Comment:
Could have something about proxy error ?
like the https://docs.centreon.com/current/en/integrations/plugin-packs/procedures/network-cisco-meraki-restapi.html#docsNav

Reply:
Added ending of documentation